c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -672,14 +672,26 @@ $(VENV_BIN)/flake8: .build/dev-requirements.timestamp
 print: $(PRINT_OUTPUT)/$(PRINT_WAR)
 
 $(PRINT_OUTPUT)/$(PRINT_WAR): $(PRINT_REQUIREMENT)
+ifeq (VENV_BIN, .build/venv/bin)
```

Comment:
`VENV_BIN` should be `$(VENV_BIN)` :-)

And you should do a rebase to make the tests working :-)